cvpysdk/subclients/vssubclient.py

```diff
@@ -85,10 +221,10 @@ def _get_subclient_content_(self):
                 children = subclient_content['children']
 
                 for child in children:
-                    path = child['path'] if 'path' in child else None
-                    display_name = child['displayName']
-                    content_type = content_types[child['type']]
-                    vm_id = child['name']
+                    path = str(child['path']) if 'path' in child else None
+                    display_name = str(child['displayName'])
```

Comment:
The methods
_get_subclient_content_
_set_subclient_content_

are using the old code.
Please update it from the latest commit from the dev branch.